uber/models/__init__.py

```diff
@@ -163,14 +163,6 @@ def default_cost(self):
                 log.exception(ex)
         return max(0, sum(values))
 
-    @property
-    def stripe_transactions(self):
```

Comment:
This would probably be better implemented as a SQLAlchemy relationship.

Reply:
It was! Sorta. Technically the relationship is to the many-to-many table (StripeTransactionAttendee), because that table's `share` column is generally relevant when we're dealing with the attendee (or group).

Reply:
I guess I should have been more explicit. I was thinking something like:
```
    stripe_transactions = relationship('StripeTransaction', backref='attendee')
```

It may take more to get that working properly. You may need a `primaryjoin` or `secondary` or something. I'll need to look more closely at the schema.